packages/ibc-utils/src/generic.rs

```diff
@@ -15,28 +15,18 @@ pub fn ibc_send_message(
     denom: String,
     amount: u128,
     memo: String,
-    timeout_height: Option<u64>,
     timeout_timestamp: Option<u64>,
     _denom_to_pfm_map: BTreeMap<String, PacketForwardMiddlewareConfig>,
 ) -> StdResult<CosmosMsg> {
     Ok(CosmosMsg::Ibc(cosmwasm_std::IbcMsg::Transfer {
         channel_id: channel,
         to_address: to,
         amount: coin(amount, denom),
-        timeout: match (timeout_height, timeout_timestamp) {
-            (Some(height), None) => IbcTimeout::with_block(IbcTimeoutBlock {
-                revision: 2,
-                height,
-            }),
-            (None, Some(timestamp)) => IbcTimeout::with_timestamp(Timestamp::from_nanos(timestamp)),
-            (Some(height), Some(timestamp)) => IbcTimeout::with_both(
-                IbcTimeoutBlock {
-                    revision: 2,
-                    height,
-                },
-                Timestamp::from_nanos(timestamp),
-            ),
-            _ => IbcTimeout::with_timestamp(env.block.time.plus_seconds(DEFAULT_TIMEOUT_TIMESTAMP)),
+        timeout: match timeout_timestamp {
+            Some(timestamp) => IbcTimeout::with_timestamp(Timestamp::from_nanos(timestamp)),
```

Comment:
does this need to be expressed in nanos? the default timestamp const is expressed in seconds (600). also feels like passing the values for this arg (`Option<u64>`) would mean passing stuff like `60000000000` to express a minute long timeout. feels like seconds are expressive enough, no?

Reply:
missed this one, yeah I think it should def be seconds here